net.sf.eclipsecs.core/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -21,21 +21,23 @@ cleanup.make_local_variable_final=true
 cleanup.make_parameters_final=true
 cleanup.make_private_fields_final=true
 cleanup.make_type_abstract_if_missing_method=false
-cleanup.make_variable_declarations_final=true
+cleanup.make_variable_declarations_final=false
 cleanup.never_use_blocks=false
 cleanup.never_use_parentheses_in_expressions=true
-cleanup.organize_imports=true
+cleanup.organize_imports=false
```

Comment:
Because the import order in the current files doesn't fit the default import order of eclipse in many cases (as mentioned in the commit message). I tried that first, and had a huge amount of changed files. So as long as I don't know what was used to create the current import order in the first place, I would not want to "fix" it to something completely different. My main issue of unused imports can be fixed without organizing them.

Reply:
Ok.